src/make_trade.ts

```diff
@@ -241,7 +284,7 @@ async function waitForTrade(
   const timeout = new Promise((resolve: (value: boolean) => void) => {
     setTimeout(resolve, TRADE_TIMEOUT_SECONDS * 1000, false);
   });
-  // Events are not very reliable, so in case we didn't receive it we query the API
+  // EVM events are not very reliable, so in case we didn't receive it we query the API
   // for the executed sell amount before concluding no trade happened.
   const sawTradeEvent = await Promise.race([traded, timeout]);
```

Comment:
Nit: Looks like this could be more appropriately named as `sawTradeOrTimedout`

Reply:
The value will be true if it `sawTradeEvent` and false if timed out (cf. the argument we pass to resolve above in setTimeout).

So I think the name is correct?